### DISCUSSION TITLE
Possible mismatch of the gpg versions on phone and computer

### DISCUSSION THREAD

**Discussion by Jimmy2027** [zeapo/Android-Password-Store#2933] [Q&A]
Hi, I have the issue that I can create passwords on the phone and read them on another machine, but if I generate the password on another machine, I cannot read it from the phone. I'm getting a OpenPGP Exception.

I guess that this issue arises because of a mismatch in the gpg versions. How can I find out what gpg version is being used by the passwordstore/ Openkeychain app?

**Comment by msfjarvis** [comment]
OpenKeychain does not use GnuPG, it implements the OpenPGP specification. Version mismatches are not a thing, you should share the full error you're seeing.

**Reply by Jimmy2027** [reply]
When tapping on a password to decrypt it, a small notification pops up:
`Error from OpenKeyChain: Encountered OpenPGP Exception during operation!`

I enabled the `Debug logging` but I'm not sure how to retrieve the log.

**Reply by msfjarvis** [reply]
The debug logging option is for Password Store's logs, not OpenKeychain's. You can follow the steps from this page but change PKG to `org.sufficientlysecure.keychain`: https://docs.passwordstore.app/docs/users/reporting-bugs

**Reply by Jimmy2027** [reply]
I'm having some issues starting adb, apparently too many open files:
`usb_libusb.cpp:911 failed to create inotify fd: Too many open files`

Will come back once I get the logs.

**Reply by Jimmy2027** [reply]
This is the log I'm getting: [log.txt](https://github.com/android-password-store/Android-Password-Store/files/14474255/log.txt)

**Reply by msfjarvis** [reply]
I don't see anything useful in the log. You might have to file an issue with OpenKeychain [here](https://github.com/open-keychain/open-keychain/issues) and see if someone there is able to help, since this is an error within OpenKeychain and [Password Store no longer uses OpenKeychain](https://github.com/android-password-store/Android-Password-Store/issues/1523).

**Reply by Jimmy2027** [reply]
Is there a doc already describing how to switch away from OpenKeychain? What is the alternative?

**Reply by Jimmy2027** [reply]
this is probably related to this OpenKeychain [issue](https://github.com/open-keychain/open-keychain/issues/2886)

**Reply by Skrilltrax** [reply]
The alternative right now is to download latest APS build from Github. It includes pgpainless so you won't need OpenKeychain.

**Reply by Jimmy2027** [reply]
the latest APS build works perfectly, thanks a lot!
